### PR TITLE
Get authenticated user

### PR DESCRIPTION
**Pull request by christopher-dG**
It seems a bit weird that this package uses the term `Owner` instead of `User`, but oh well.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Presumably that's because an owner can be a user or an organization?

**Comment by christopher-dG** [comment]
True, it's just not good for my tab-completion-powered API exploration 😅 

Windows nightly is failing, due to some unrelated issue. Should probably allow failure on those.

**Comment by DilumAluthge** [comment]
Seems fine to allow failures on Windows nightly.

File: src/owners/owners.jl [hunk 1]
```diff
@@ -44,6 +44,11 @@ isorg(owner::Owner) = something(owner.typ, "") == "Organization"
 
 @api_default owner(api::GitHubAPI, owner_obj::Owner; options...) = owner(api, name(owner_obj), isorg(owner_obj); options...)
 
+@api_default function owner(api::GitHubAPI; options...)
```
Comment (fredrikekre) [inline src/owners/owners.jl:47]: `whoami` would be a nice name for this method.

**Comment by christopher-dG** [comment]
Renamed to `whoami`. The README changes are big but I just aligned the tables automatically with my editor. The only line I added is the `whoami` line [here](https://github.com/JuliaWeb/GitHub.jl/blob/56c9aa23c593d76ae345deb8377ef7158cfd5798/README.md#users-and-organizations).

File: .travis.yml [hunk 2]
```diff
@@ -9,12 +9,16 @@ julia:
...
-    
+
 branches:
     only:
         - master
         - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
+jobs:
+    allow_failures:
+        - julia: nightly
```
Comment (DilumAluthge) [inline .travis.yml:20]:
```suggestion
        - julia: nightly
          os: windows
```

**Review by DilumAluthge — CHANGES_REQUESTED** [review]
I would suggest only allowing failures on Julia nightly on Windows.

File: .travis.yml (hunk 2 above)
Comment (christopher-dG) [reply .travis.yml:20]: I'm usually of the opinion that nightly can fail on any OS it wants, but this is fine too.

**Review by DilumAluthge — APPROVED** [review]
LGTM.

**Pull request merged**